go.mod

```diff
@@ -1,4 +1,4 @@
-module go.dedis.ch/kyber/v3
+module go.dedis.ch/kyber/v4
 
 go 1.18
```

Comment:
have we considered bumping the go version to something newer? does it break things?

Reply:
we actually have upgraded it from Go 1.13 to Go 1.18 already! 
In truth it would probably be nice to update to Go 1.22 to get the new Loop behaviour, but at the same time a library like Kyber kinda wants to support older codebases too I guess.